src/main/kotlin/CarController.kt

```diff
@@ -0,0 +1,39 @@
+import kotlin.random.Random
+
+const val RANDOM_RANGE = 10
+class CarController(carCount: Int, private val playCount: Int) {
+    private var cars: List<Car>
+
+    init {
+        cars = makeCars(carCount)
+    }
+
+    private fun makeCars(count: Int): List<Car> {
+        val cars = mutableListOf<Car>()
+        for (i in 0 until count) {
```

Comment:
`repeat()`를 찾아보고 그 기능을 사용해 보면 어떨까요?

```suggestion
        repeat(count) {
```

Reply:
[e95d594](https://github.com/next-step/kotlin-racingcar/pull/80/commits/e95d594f015a94871e57b1b8c833e17b9fe4782b) 수정 반영 하였습니다.